src/com/facebook/buck/ide/intellij/lang/java/JavaLibraryModuleRule.java

```diff
@@ -48,6 +48,7 @@ public void apply(
     addDepsAndSources(target, true /* wantsPackagePrefix */, context);
     JavaLibraryRuleHelper.addCompiledShadowIfNeeded(projectConfig, target, context);
     context.setJavaLanguageLevel(JavaLibraryRuleHelper.getLanguageLevel(projectConfig, target));
+    context.setCompilerOutputPath(moduleFactoryResolver.getCompilerOutputPath(target));
```

Comment:
Can you please add this one line change to kotlin and groovy module rules as well